lib/SILOptimizer/Transforms/SILCodeMotion.cpp

```diff
@@ -32,6 +32,8 @@
 #include "llvm/Support/CommandLine.h"
 #include "llvm/Support/Debug.h"
 
+#include <iostream>
```

Comment:
Please remove this and the print statement.

Reply:
@zoecarver  More meta, in general we do not use iostream in LLVM based projects. I would familiarize oneself with these two documents: 

1. https://www.llvm.org/docs/ProgrammersManual.html.
2. https://www.llvm.org/docs/CodingStandards.html

For instance the iostream issue is here: https://www.llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden.

Reply:
@zoecarver  More meta, in general we do not use iostream in LLVM based projects. I would familiarize oneself with these two documents: 

1. https://www.llvm.org/docs/ProgrammersManual.html.
2. https://www.llvm.org/docs/CodingStandards.html

For instance the iostream issue is here: https://www.llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden.

Reply:
Thanks for the links. Forgot to remove it before pushing my code. As I said, this is very much a work in progress.

Reply:
@zoecarver No worries! = ).

Reply:
The only difference with the coding standards is that we generally make functions and variables camelCase, not CamelCase. (As always, match the style of the code around it, but if you are writing new code, new functions, plz do it this way = ).

Reply:
I will try to do that. I have seen a lot of TitleCase variables. Should variables be TitleCase or camelCase?

Reply:
camelCase. We have been doing the change slowly over time in new code.